addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -359,6 +363,17 @@ public Action Command_ToggleHints(int client, int args)
 	return Plugin_Handled;
 }
 
+public Action Command_PRinfo_help(int client, int args){
+
+	if(IsValidClient(client)){
+		PrintToConsole(client, "%t", "PRinfo_help");
```

Comment:
Personally I would prefer `%T`, it'll use the client language instead of the server language. `%T` requires the client index after the translation name, in this case `PrintToConsole(client, "%T", "PRinfo_help", client);`.

You can find more [here](https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#Usage_in_a_Plugin).